build.gradle

```diff
@@ -60,8 +60,8 @@ dependencies {
     
     testCompile group: 'junit', name: 'junit', version: '4.+'
 
-    compile group: 'eu.mihosoft.ext.org.fxyz', name: 'extfxyz', version: '0.4'
-    compile group: 'eu.mihosoft.ext.org.fxyz', name: 'extfxyz', version: '0.4', classifier: 'sources'
+    //compile group: 'eu.mihosoft.ext.org.fxyz', name: 'extfxyz', version: '0.4'
```

Comment:
I removed the fxyz as a dependency, because extfxyz depends on a version of JCSG, causing the inclusion of outdated classes and conflicts in the classpath whenever this library is included in a project.